automated-tests/src/test/resources/cucumber/network/fcNetwork.feature

```diff
@@ -49,6 +49,17 @@ Feature: In order to manage FC Networks
       And OneView gets Resource by ID
     Then I get a Resource Name
 
+  @patch
+  Scenario: Update an Fc Network by Patch
+    Given name "fc-network-bdd" for Resource
+      And Resource values will be updated as follows:
+      | op    | replace      |
+      | path  | /scopeUris   |
+      | value | Sample Scope |
+    When OneView gets Resource by Name
+      And OneView runs Fc Network patch
+    Then Resource is found
```

Comment:
I think we should change the verification for a success status, or to verify the resources properties (scopeUris attribute).